karapace/avro_compatibility.py

```diff
@@ -2,13 +2,15 @@
     ARRAY, ArraySchema, BOOLEAN, BYTES, DOUBLE, ENUM, EnumSchema, Field, FIXED, FixedSchema, FLOAT, INT, LONG, MAP,
     MapSchema, NamedSchema, Names, NULL, RECORD, RecordSchema, Schema, SchemaFromJSONData, STRING, UNION, UnionSchema
 )
-from enum import Enum
-from typing import Any, cast, Dict, List, Optional, Set
+from enum import Enum, unique
+from typing import Any, cast, Dict, Generic, List, Optional, Set, TypeVar
 
 import json
 
+E = TypeVar("E", bound=Enum)
 
-def parse_json_ignore_trailing(s: str) -> Any:
+
+def parse_avro_schema_definition(s: str) -> Any:
```

Comment:
Can we narrow down Any, to something a bit more specific?

Reply:
Done :)